client/lib/oauth-store/test/index.js

```diff
@@ -1,26 +1,25 @@
 /**
  * External dependencies
  */
-
 var Dispatcher = require( 'dispatcher' ),
 	expect = require( 'chai' ).expect,
 	sinon = require( 'sinon' );
 
 /**
  * Internal dependencies
  */
-var oAuthToken = require( 'lib/oauth-token' ),
-	testConstants = require( '../constants' );
+import { actions } from '../constants';
+import * as oAuthToken from 'lib/oauth-token';
```

Comment:
Fixed, test that uses fake dom is a bit slower, but much more clean now :) Let me know if that makes sense.